### ISSUE TITLE
whether logits self KD can be used alone

### ISSUE THREAD

**Issue by kwk12306** [winycg/Self-KD-Lib#1]
Hi Wingcg!
         Your idea has inspired me a lot. May I ask whether logits self KD can be used alone, instead of combining logits self KD and feature self KD, because the size of the model and the amount of calculation are taken into account.Thanks a lot!

**Comment by winycg** [comment]
Thanks for your attention of our work. It is OK to apply logits self KD alone, but the performance may drop a lot.
